### PR TITLE
Improve example for get_area_def

### PR DESCRIPTION
**Pull request by gerritholl**
Improve description and add example in the documentation for get_area_def.

 - [ ] Closes #xxxx 
 - [x] Fully documented 
 - [x] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

**Comment by gerritholl** [comment]
Oh, I see that this overlaps a lot with #1019.

**Comment by gerritholl** [comment]
@djhoese Maybe you want to merge this into the work you are doing on #1019.  As far as I can tell that one does not update the description of `get_area_def`.

File: doc/source/dev_guide/custom_reader.rst [hunk 1]
```diff
@@ -431,8 +432,15 @@ On top of that, two attributes need to be defined: ``start_time`` and
...
-            # TODO
-            pass
+            return pyresample.geometry.AreaDefinition(
+                "some_area_name",
+                "on-the-fly area",
+                "geos",
+                "+a=6378169.0 +h=35785831.0 +b=6356583.8 +lon_0=0 +proj=geos",
+                3636,
+                3636,
+                np.array([-5456233.41938636, -5453233.01608472, 5453233.01608472, 5456233.41938636]))
```
Comment (djhoese) [inline doc/source/dev_guide/custom_reader.rst]: It is probably best (to reduce confusion) to not have this be a numpy array. Don't worry about it right now though.

Comment (mraspaud) [reply doc/source/dev_guide/custom_reader.rst]:
```suggestion
                [-5456233.41938636, -5453233.01608472, 5453233.01608472, 5456233.41938636])
```

**Comment by djhoese** [comment]
I've included the stuff from here in my PR. This should close automatically when that is merged.

**Pull request closed without merging**